unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/Transform/System/ECSTransformParentingSystem.cs

```diff
@@ -1,7 +1,10 @@
+using DCL.ECS7.InternalComponents;
 using DCL.Models;
 
 public static class ECSTransformParentingSystem
 {
+    public static IInternalECSComponent<InternalSceneBoundsCheck> internalSceneBoundsCheckComponent;
+
     public static void Update()
```

Comment:
we can avoid that `public static field`

```suggestion
    public static void Create(IInternalECSComponent<InternalSceneBoundsCheck> internalSceneBoundsCheckComponent) {
          return ()=> Update(internalSceneBoundsCheckComponent);
    }
    private static void Update(IInternalECSComponent<InternalSceneBoundsCheck> internalSceneBoundsCheckComponent)
```